Cargo.toml

```diff
@@ -19,19 +19,22 @@ maintenance = { status = "actively-developed" }
 
 [dependencies]
 
-async-stream = "^0"
+#async-stream = "^0"
 futures = { version = "^0", default-features = false, features = ["alloc"] }
 log = "^0"
 env_logger="^0"
+tokio = { version = "0.2.22", features = ["macros"] }
```

Comment:
This should be `^0.2` (to avoid versions `0.3.x` which are not fully compatible).